samples/server/petstore/rust-server/output/petstore-with-fake-endpoints-models-for-testing/api/openapi.yaml

```diff
@@ -726,6 +726,7 @@ paths:
               properties:
                 integer:
                   description: None
+                  format: int32
```

Comment:
FYI. Our petstore (fake) spec does not specify the format. I think swagger parser now defaults format to int32 for integer type if no format is specified.

Not sure if the community welcomes this change. We'll see.